jekyll-assets/_layouts/boxes.html

```diff
@@ -34,6 +34,31 @@ <h1><a href="/documentation/"><b>Raspberry Pi</b> Documentation</a></h1>
       </section>
     </div>
 
+    <div id="related" role="complementary">
+      <div id="related-title">
+        <h2>Related resources</h2>
+      </div>
+      <div id="related-container" role="complementary">
+      <section id="related-content">
```

Comment:
Should this `<section>` (and its corresponding closing tag on line 58) be indented one level more?